packages/cli/src/constants.ts

```diff
@@ -76,5 +76,7 @@ export enum CLILogLevel {
 export const sqlPasswordQustionName = "sql-password";
 
 export const sqlPasswordConfirmQuestionName = "sql-confirm-password";
-
 export const deployPluginNodeName = "deploy-plugin";
+
+export const connection = "connection";
```

Comment:
"connection" and "close" are built-in event type and is defined by node-http APIs, ok to directly use without defining constants... just for code simplify.

``` node
// net.d.ts
on(event: "close", listener: () => void): this;
on(event: "connection", listener: (socket: Socket) => void): this;
```

Reply:
Remove inappropriate constants.